seatunnel-core/pom.xml

```diff
@@ -30,6 +30,17 @@
     <artifactId>seatunnel-core</artifactId>
     <packaging>pom</packaging>
 
+    <modules>
+        <module>seatunnel-core-base</module>
+        <module>seatunnel-core-flink</module>
+        <module>seatunnel-core-spark</module>
+        <module>seatunnel-core-flink-sql</module>
+        <module>seatunnel-core-starter</module>
+        <module>seatunnel-flink-starter</module>
+        <module>seatunnel-spark-starter</module>
+        <module>seatunnel-starter</module>
+    </modules>
```

Comment:
Should the engine-all profile be removed?
@CalvinKirs CC

Reply:
> Should the engine-all profile be removed? @CalvinKirs CC

Yes, `engine-all` profile is not used now.